sdks/python/build.gradle

```diff
@@ -226,6 +228,26 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
   }
 }
 
+task precommitIT(dependsOn: ['installGcpTest', 'sdist']) {
```

Comment:
sdist should run before installGcpTest. I believe gradle has a rule for that.

Reply:
done